### DISCUSSION TITLE
chore(deps) isn't added any longer to commit message

### DISCUSSION THREAD

**Discussion by chkpnt** [renovatebot/renovate#20535] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

renovate/renovate:34.97.6 (Docker)

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

GitLab EE 15.7.5

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Renovate doesn't add the prefix "chore(deps)" anymore despite `:semanticCommitTypeAll(chore)` is configured. Three weeks ago, it worked fine. Since around two weeks, the prefix is missing.

I haven't changed the version of renovate I'm using (34.97.6), so I can't explain the changed behaviour.

### Relevant debug logs

I don't see anything relevant in the log. 

The repository is read as configured:

```
{"name":"renovate","hostname":"runner-vz1bzss-project-49-concurrent-0","pid":15,"level":20,"logContext":"hkRRzHeC0QWuFGDmc0uJE","repository":"<redacted-repo-name>","fileName":"renovate.json","config":{"extends":["config:base",":semanticCommitTypeAll(chore)"],"assignees":["dschung"],"schedule":["every weekend"],"packageRules":[{"matchManagers":["npm"],"matchDepTypes":["dependencies"],"matchUpdateTypes":["minor","patch"],"groupName":"dependencies (GUI)"},{"matchManagers":["npm"],"matchDepTypes":["devDependencies"],"matchUpdateTypes":["minor","patch"],"groupName":"devDependencies (GUI)"},{"matchManagers":["ruby-version"],"enabled":false}],"npm":{"stabilityDays":3}},"msg":"Repository config","time":"2023-02-20T10:03:23.651Z","v":0}
```

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by HonkingGoose** [comment]
Here's the same log message, but easier to read:

```json
{
  "name": "renovate",
  "hostname": "runner-vz1bzss-project-49-concurrent-0",
  "pid": 15,
  "level": 20,
  "logContext": "hkRRzHeC0QWuFGDmc0uJE",
  "repository": "<redacted-repo-name>",
  "fileName": "renovate.json",
  "config": {
    "extends": ["config:base", ":semanticCommitTypeAll(chore)"],
    "assignees": ["dschung"],
    "schedule": ["every weekend"],
    "packageRules": [
      {
        "matchManagers": ["npm"],
        "matchDepTypes": ["dependencies"],
        "matchUpdateTypes": ["minor", "patch"],
        "groupName": "dependencies (GUI)"
      },
      {
        "matchManagers": ["npm"],
        "matchDepTypes": ["devDependencies"],
        "matchUpdateTypes": ["minor", "patch"],
        "groupName": "devDependencies (GUI)"
      },
      { "matchManagers": ["ruby-version"], "enabled": false }
    ],
    "npm": { "stabilityDays": 3 }
  },
  "msg": "Repository config",
  "time": "2023-02-20T10:03:23.651Z",
  "v": 0
}
```

**Comment by rarkins** [comment, marked as answer]
This config controls which _type_ of semantic commit is used: https://docs.renovatebot.com/presets-default/#semanticcommittypeallarg0

But you might also need to set `"semanticCommits": true` instead of `auto`: https://docs.renovatebot.com/configuration-options/#semanticcommits

**Comment by chkpnt** [comment]
Thank you, adding `"semanticCommits": "enabled"` did indeed restore the previous behaviour. :-)
